engine/src/test/java/io/zeebe/engine/processor/workflow/ExpressionProcessorCorrelationKeyResultHandlerTest.java

```diff
@@ -23,9 +23,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class ExpressionProcessorCorrelationKeyResultHandlerTest {
 
-  public static final String TEST_EXPRESSION = "expression";
+  private static final String TEST_EXPRESSION = "expression";
+  private static final long VARIABLE_SCOPE_KEY = 21;
```

Comment:
I'm still not really sure about this. The name implies something about the value, but it still seems like a magic number. Why is it 21?

Reply:
The variable scope key is not used in the test but it needs to be passed into the context. So, I chose a random number :)